### ISSUE TITLE
a very Seriou bug

### ISSUE THREAD

**Issue by SKT01KP** [lightvector/KataGo#457]
![A334C0483A9241210EB0DE3660A0A96D](https://user-images.githubusercontent.com/64546333/113433847-8e3a3b80-93a5-11eb-8a0e-631fc476253f.jpg)
I put katago in charge,but it lost in just five steps

**Comment by Friday9i** [comment]
With so few information, I doubt anyone can help... Normally Sabaki gives much more details when it runs an engine, that should help understand what happens. I guess the window means it resigned, but it should not so early in the game.
Try to reinstall KataGo by following the instructions (https://github.com/lightvector/KataGo#how-to-use), and if that still doesn't work, give more details on what you get so that we can help!

**Comment by lightvector** [comment]
Right, if you think something's going wrong, try looking in the gtp log file for a error message and/or post the error or the whole gtp log here, or try running KataGo exe directly on the command line. You can try the `benchmark` command to see if that works, you can also run the `gtp` command and type in `showboard` and `genmove b` and `genmove w` to play moves manually one by one and see if it crashes (this is basically what Sabaki is doing).

**Comment by Saturday-morning** [comment]
I don't know too,
but I also see the photo above by chance.
https://www.bilibili.com/read/cv10598114

**Comment by lightvector** [comment]
So, either his configuration or machine has a bug, or here's an alternate guess: because he forced KataGo to pass on the first turn, that creates a completely losing position, so KataGo resigns, the same way it would resign after a few turns in any other completely losing position.

KataGo doesn't resign in handicap games or games where the komi is set to be disfavorable, because the game itself is known to be a handicap game or a reverse-komi game, so being behind is expected. However, this is not such a case - this is is probably an *even* game, and if in an even game you find yourself behind massively, you can resign.

**Comment by lightvector** [comment]
As for why pass on the first turn - no idea, but if you think Sabaki has a bug, then you need to figure out what you did wrong with Sabaki, or else use some other GUI instead that you find less confusing. Or, dig in to the log files and post the relevant logs so that actual things can be investigated.

I suspect this isn't a "very Seriou bug" - there's a good chance there was just some misclick that resulted in confusion about how to use Sabaki, so that you got Sabaki into a state where the game was paused or stuck and didn't know how to get it unstuck, so then you have the pass first, and after that, without actually seeing the logs and the configuration and only just taking a guess, I suspect everything is normal behavior. (By the way, you can specify the resign behavior in the config file, if you like).

**Comment by HackYardo** [comment]
Well, it is a "**fake bug**". 
Question 1 KataGo resigns at move 5:
Black passes at move 1=>Black losses score=>reaches resign-threshold=>resigns
Question 2 Sabaki can not play black at move 1:
Sabaki has a command called "generate move" to let engine play
![image](https://user-images.githubusercontent.com/57677458/113513876-40d4ee80-959e-11eb-8eb3-c683908e66c6.png)

(If you are a chinese, you can switch to chinese in Sabaki:
![image](https://user-images.githubusercontent.com/57677458/113513863-3581c300-959e-11eb-9ca8-59b25861ec85.png)
)
